src/react-components/home-root.js

```diff
@@ -275,7 +275,55 @@ class HomeRoot extends Component {
                   </div>
                 </div>
               )}
+              <IfFeature name="show_newsletter_signup">
+                <div id="mc_embed_signup" className={styles.footerContent}>
+                  <form
+                    action="https://mozilla.us19.list-manage.com/subscribe/post?u=a97038e0879516f7b8300d67f&amp;id=ebf17d04e4"
+                    method="post"
+                    id="mc-embedded-subscribe-form"
+                    name="mc-embedded-subscribe-form"
+                    className="validate"
+                    target="_blank"
+                    noValidate
+                  >
+                    <div id="mc_embed_signup_scroll">
+                      <div className={styles.subscribeText}>
+                        <FormattedMessage id="home.subscribe_to_mailing_list" />
+                      </div>
+                      <div className={styles.subscribeBar}>
+                        <div className="mc-field-group">
+                          {/* <label htmlFor="mce-EMAIL">Email Address </label> */}
+                          <input
+                            placeholder="Enter email address.."
+                            type="email"
+                            name="EMAIL"
+                            className={styles.emailInput}
+                            id="mce-EMAIL"
+                          />
+                        </div>
+                        <div id="mce-responses" className="clear">
+                          <div className="response" id="mce-error-response" style={{ display: "none" }} />
+                          <div className="response" id="mce-success-response" style={{ display: "none" }} />
+                        </div>
+                        <div style={{ position: "absolute", left: "-5000px" }} aria-hidden="true">
+                          <input type="text" name="b_a97038e0879516f7b8300d67f_ebf17d04e4" tabIndex="-1" value="" />
+                        </div>
```

Comment:
What is this for? Do we need it?

Reply:
This was to guard against Robots, but I'm not sure how. May only work if you add their javascript to the page, too.